app/views/verify/phone/new.html.slim

```diff
@@ -20,4 +20,8 @@ p = t('idv.messages.phone.same_as_2fa')
   = f.button :submit, t('forms.buttons.continue'), class: 'btn btn-primary btn-wide'
 = render 'shared/cancel', link: verify_cancel_path
 
-== javascript_include_tag 'misc/verify-modal'
+- if @modal
```

Comment:
Yeah I agree with @jessieay here, I expect `@modal` to be a modal object type thing. As currently written this seems more like `@modal_type` to me